cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -36,19 +36,40 @@ Feature: Users can authneticate with OIDC authenticator
       member: !user alice
 
     - !user alice@conjur.net
-
-    - !grant
-      role: !group conjur/authn-oidc/keycloak/users
-      member: !user alice@conjur.net
     """
     And I am the super-user
     And I successfully set OIDC variables
 
   Scenario: A valid id token to get Conjur access token
-    When I successfully authenticate via OIDC
+    When I successfully authenticate via OIDC with id token:
+    """
+    {"preferred_username": "alice","email": "alice@conjur.net"}
+    """
+
     Then "alice" is authorized
 
+    # Verify the returned access token is valid for retrieving a secret
+    Given a policy:
```

Comment:
Usually you would put all of the "Given" blocks at the beginning of a cucumber test. Cucumber tests generally follow a `Given`-`When`-`Then` flow (with `And`s inbetween if needed). It's easier to reason about them this way.

Reply:
done